nautobot/core/management/commands/audit_dynamic_groups.py

```diff
@@ -0,0 +1,33 @@
+from django.core.management.base import BaseCommand
+
+from nautobot.core.utils.lookup import get_filterset_for_model
+
+
+class Command(BaseCommand):
+    help = "Audit all existing DynamicGroup instances in the database and output invalid filter data"
+
+    def handle(self, *args, **options):
+        from nautobot.extras.models import DynamicGroup
+
+        print("\n>>> Auditing existing DynamicGroup data for invalid filters ...\n")
```

Comment:
In management commands, it's [recommended](https://docs.djangoproject.com/en/4.2/howto/custom-management-commands/) to use `self.stdout.write()` instead of `print()`.